app/scripts/directives/nav.js

```diff
@@ -242,11 +250,16 @@ angular.module('openshiftConsole')
         };
 
         $scope.closeOrderingPanel = function() {
-          _.set($scope, 'ordering.panelName', "");
+          $scope.orderingPanelVisible = false;
         };
 
         $scope.showOrderingPanel = function(panelName) {
-          _.set($scope, 'ordering.panelName', panelName);
+          $scope.orderingPanelVisible = true;
+          $scope.orderKind = panelName;
+        };
+
+        $scope.onSearchToggle = function(showMobileSearch) {
+          _.set($rootScope, 'view.hasProjectSearch', showMobileSearch);
```

Comment:
Just curious what happens if I navigate to a different page with the search open, either at mobile or desktop. For instance, if I click the back button. (Not sure what the right behavior is.)

Reply:
When you come back, the search is still open.